confluent_kafka/avro/cached_schema_registry_client.py

```diff
@@ -269,7 +269,8 @@ def test_compatibility(self, subject, avro_schema, version='latest'):
             else:
                 log.error("Unable to check the compatibility")
                 False
-        except:
+        except Exception as e:
+            log.error("_send_request() failed: %s", e)
```

Comment:
use `log.exception()`... it is the same as `log.error()` but it will automatically append the exception:
 
    log.exception("_send_request() failed.")